src/configurePipeline/utility/pipelineHelper.ts

```diff
@@ -0,0 +1,60 @@
+import * as fs from "fs";
+import * as path from "path";
+
+import * as vscode from 'vscode';
+import { PipelineTargets } from "../model/Common";
+
+export async function getSelectedPipeline(nodeUri: vscode.Uri[], dockerUri: vscode.Uri[]): Promise<string> {
```

Comment:
IMO this is not a helper/utility function as it is has UX interaction. Should we separate the UX wizard steps from helpers and services files ?